pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -85,23 +85,46 @@ all: manager
 # More info on the awk command:
 # http://linuxcommand.org/lc3_adv_awk.php
 
-.PHONY: help
-help:  ## Display this help.
+help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
+##@ Development
+
+manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResource definition objects.
```

Comment:
```suggestion
manifests: controller-gen ## ## Generate manifests e.g. CRD, RBAC, etc..
```
I prefer less verbose and more generifc info as it is now.